tests/bud.bats

```diff
@@ -43,6 +43,19 @@ load helpers
   expect_output --substring "unsupported value"
 }
 
+@test "build with inline default RUN --network=default" {
+  _prefetch alpine
+  run readlink /proc/self/ns/net
+  hostns=$output
+  run_buildah build --network=host $WITH_POLICY_JSON -t source -f $BUDFILES/inline-network/Dockerfile4
+  firstns=${lines[2]}
+  assert "${hostns}" == "$firstns"
+  run_buildah build --network=private $WITH_POLICY_JSON -t source -f $BUDFILES/inline-network/Dockerfile4
+  secondns=${lines[2]}
+  assert "$secondns" != "$firstns"
+  assert "$secondns" != "$hostns"
```

Comment:
We only need to do one of these tests if we've already verified that "$firstns" and "$hostns" are the same.